modules/ossm-jaeger-supported-configurations.adoc

```diff
@@ -0,0 +1,44 @@
+////
+This REFERENCE module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-supported-configuration_{context}"]
+= Jaeger supported configuration
+
+For the {ProductVersion} release of {ProductName}the only supported configuration of Jaeger is as follows:
+
+.Supported Jaeger configurations
```

Comment:
Is this documenting generic Jaeger capabilities, or those configured through ServiceMeshControlPlane, as the the latter only exposes a subset.

Reply:
I had a long Slack conversation with @brian-avery tonight and I think I finally understand how the Maistra operator and SMCP affect Jaeger.

Reply:
Agree, this should only be dealing with the config of Jaeger that is exposed via the Maistra CRD.